openstudiocore/src/project/Test/ProjectFixture.cpp

```diff
@@ -88,7 +88,11 @@ openstudio::project::ProjectDatabase ProjectFixture::getDatabase(
 void ProjectFixture::removeAllDatabases() {
   for (const openstudio::path& dbDir : databaseDirs) {
     if (boost::filesystem::exists(dbDir)) {
-      boost::filesystem::remove_all(dbDir);
+      try{
```

Comment:
This was failing and causing several tests to fail.  I did not understand why, but it was causing unit test failures